network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html

```diff
@@ -3,14 +3,18 @@
 {% load wagtailcore_tags mini_site_tags %}
 
 {% block subcontent %}
-<div class="py-4 py-md-5 col-lg-8 offset-lg-2">
-  <div class="cms">
-    <h1 class="h1-heading">{{ page.title }}</h1>
-    <p class="h6-heading mb-5">By {{ page.author }} | {% if page.first_published_at %}{{ page.first_published_at|date:"F j, Y" }}{% else %}not published yet{% endif %}</p>
-    <div class="blog-body">
-      {{ page.body }}
+  <div class="py-4 py-md-5 col-lg-8 offset-lg-2">
+    <div class="cms">
+      <h1 class="h1-heading">{{ page.title }}</h1>
+      <p class="h6-heading mb-5">By {{ page.author }} | {% if page.first_published_at %}{{ page.first_published_at|date:"F j, Y" }}{% else %}not published yet{% endif %}</p>
+      <div class="blog-body">
+        {{ page.body }}
+      </div>
     </div>
+    {% include "./fragments/post_tags.html" %}
```

Comment:
Note that this bit needs to be removed prior to landing!

Reply:
turned this fragment into a code-comment generator, so it's there, but only in view-source